modules/oe_translation_poetry/src/Plugin/tmgmt/Translator/PoetryTranslator.php

```diff
@@ -376,24 +389,33 @@ public function contentTranslationOverviewAlter(array &$build, RouteMatchInterfa
     $request_type = $this->getRequestType($entity);
 
     // If we have accepted languages in Poetry, we need to include them (and
-    // some others automatically in a potential update request).
+    // some others automatically in a potential update request). Also, these
+    // are needed to determine which languages should be omitted from a
+    // language addition request.
     // @see oe_translation_poetry_form_tmgmt_content_translate_form_alter().
-    if ($request_type->getType() === PoetryRequestType::UPDATE) {
-      $build['#accepted_languages'] = $accepted_languages;
-      $build['#translated_languages'] = $translated_languages;
-      // Load the completed jobs in the same request. For this we need to get
-      // the request ID from one of the accepted jobs in order to filter the
-      // completed jobs and include only the ones in this request. For the
-      // translated jobs we don't have to filter by request ID because those
-      // still contain the `poetry_state` value which indicates they are part
-      // of the last request for that content.
+    $build['#accepted_languages'] = $accepted_languages;
+    $build['#translated_languages'] = $translated_languages;
+    $build['#completed_languages'] = [];
+    // Load the completed jobs in the same request. For this we need to get
+    // the request ID from one of the accepted jobs in order to filter the
+    // completed jobs and include only the ones in this request. For the
+    // translated jobs we don't have to filter by request ID because those
+    // still contain the `poetry_state` value which indicates they are part
+    // of the last request for that content.
+    if ($accepted_languages) {
       $accepted_job_info = reset($accepted_languages);
       /** @var \Drupal\tmgmt\JobInterface $job */
       $job = $this->entityTypeManager->getStorage('tmgmt_job')->load($accepted_job_info->tjid);
       $completed_languages = $this->getCompletedJobsByLanguage($entity, $job);
```

Comment:
I know it was not introduced in this PR but this method shouldn't get the $job as a parameter, it should get the request_id.